wifiphisher/pywifiphisher.py

```diff
@@ -1023,8 +1036,10 @@ def run():
 
     clients_APs = []
     APs = []
-    args.accesspoint = ap_mac
-    args.channel = channel
+
+    if not (args.essid and args.channel):
+        args.accesspoint = ap_mac
```

Comment:
`args.accesspoint` is used to deauth the target AP. In this scenario, we don't have a target AP. We want to deauth every single AP around us. Set this to `None` instead.